phoenix-core/src/it/java/org/apache/phoenix/end2end/index/ImmutableIndexIT.java

```diff
@@ -417,6 +422,58 @@ public void testGlobalImmutableIndexDelete() throws Exception {
         }
     }
 
+    @Test
+    public void testGlobalImmutableIndexFullWriteOnLast() throws Exception {
```

Comment:
Nit: This test only validates phase1 that it does not write the full row and only unverified. The test name should probably be something like "testGlobalImmutableIndexUnverifiedOnlyInPhase1"